src/redux/selectors/sources/sources.js

```diff
@@ -11,7 +11,13 @@ export const sourcesMetaSelector = state => state.sources.meta;
 
 export const sourcesDataSelector = state => state.sources.data;
 
-export const getDescription = ({ api, owner, repo, path }) => {
+export const getDescription = source => {
+  const { url, api, owner, repo, path } = source;
+  if (url) {
+    return url;
+  }
+  // This is for backwards compat...prob irrelevant at this point.
+  // could just change the redux:persist key for now.
```

Comment:
fetch-source doesn't support backwards compat so will instead clear down storage (change key, and delete old key, unless there's a better way to do it when releasing new versions...).

Reply:
This has been handled with redux-persist migrations